tools/xtask/src/main.rs

```diff
@@ -82,7 +83,16 @@ async fn generate_rest_api_doc() -> anyhow::Result<()> {
     let join_handle = tokio::spawn(meta_service.run(shutdown_watch, Mock));
 
     let res = RetryPolicy::fixed_delay(Duration::from_millis(100), 20)
-        .retry_operation(|| async { reqwest::get(openapi_address.clone()).await?.text().await })
+        .retry_operation(|| async {
+            reqwest::Client::builder()
+                .build()?
+                .get(openapi_address.clone())
+                .header(ACCEPT, "application/json")
+                .send()
+                .await?
+                .text()
+                .await
```

Comment:
Would be great to state in the commit message why the openapi generation needs the `ACCEPT` header. Syntactically this change looks good just that I don't know why some things won't be generated w/o this header.

Reply:
this looks like a bug in okapi-operation, which i may have introduced :). at least now is generating the right openapi. Will add to the commit message.